src/outline/manager.js

```diff
@@ -1,14 +1,13 @@
 /* -*- coding: utf-8-dos -*-
    Outline Mode Manager
 */
-var panes = require('pane-registry')
+const panes = require('pane-registry')
```

Comment:
These changes seem unrelated as well?
We can do this kind of cleanup at the same time as moving this file to typescript, adding unit tests for it, fixing up its documentation, and adding examples for how to use the functions it exports.